host.py

```diff
@@ -557,7 +557,7 @@ def bf_pxeboot(self, nfs_iso: str, nfs_key: str) -> Result:
 
     def bf_firmware_upgrade(self) -> Result:
         logger.info("Upgrading BF firmware")
-        return self.run_in_container("/fwup")
+        return self.run_in_container("/fwup -v 24.39.2048")
```

Comment:
Please add a comment here saying why we have this temporary W/A.

Reply:
Good point, updated.